mock-core-configs/etc/mock/amazonlinux-2023-x86_64.cfg

```diff
@@ -0,0 +1,5 @@
+include('templates/amazonlinux-2023.tpl')
+
+config_opts['root'] = 'amzn2023'
+config_opts['target_arch'] = 'x86-64'
+config_opts['legal_host_arches'] = ('x86-64',)
```

Comment:
You've got a typo here, it's `x86_64`

Reply:
Well, you can tell what architecture my main dev platforms are! Will fix :)